HEEDLS-542 Replace break-word with break-all

AlexJacksonDS opened the pull request:
Changed all usages of word-break: break-word to word-break: break-all

IE screens the ticket was raised for:
![image](https://user-images.githubusercontent.com/59561751/124098795-ca3c7580-da54-11eb-8ac7-df4460de2614.png)

![image](https://user-images.githubusercontent.com/59561751/124098827-d1638380-da54-11eb-8efd-74389bd33b91.png)

stellake commented on DigitalLearningSolutions.Web/Styles/index.scss:
```diff
@@ -131,7 +131,7 @@ h1#page-heading {
 .nhsuk-summary-list__value {
   @include govuk-media-query($until: tablet) {
-    word-break: break-word;
+    word-break: break-all;
```
Doesn't break-all break the second word instead of putting it on a new line when there's two words in one line?

Example here:
https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

AlexJacksonDS commented:
I've added the IE only fix using the same pattern we use elsewhere in the css.

Since there was only one instance of each of the other css classes with word-break being used, I've instead added the .word-break css class to the html elements and removed it from the specific css classes so that we don't have to define the IE fix multiple times.

stellake commented on DigitalLearningSolutions.Web/Styles/index.scss:
```diff
@@ -131,7 +131,7 @@ h1#page-heading {
 .nhsuk-summary-list__value {
   @include govuk-media-query($until: tablet) {
-    word-break: break-word;
+    word-break: break-all;
```
Ah - forgot to change here. Also, does the hack work ok on other main browsers?

AlexJacksonDS replied:
Yes, this adds an extra declaration for IE only, other browsers only see break-word. IE gets both and ignores the one it doesn't support.

AlexJacksonDS replied:
There are too many instances of this to add a css class to each of them so I've defined it using a mixin instead

stellake reviewed (approved):
Very nice - looks good! 👍
